modules/tensor_mechanics/src/materials/ComputeLinearElasticStress_CUT_DT.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeLinearElasticStress_CUT_DT.h"
+
+template<>
+InputParameters validParams<ComputeLinearElasticStress_CUT_DT>()
+{
+  InputParameters params = validParams<ComputeStressBase>();
+  params.addClassDescription("Compute stress using elasticity for small strains");
+  return params;
```

Comment:
Another option would be to have a `modules/tensor_mechanics/src/materials/test/` subdirectory. But I guess this test can be done without involving TM at all.